docker-compose.yml

```diff
@@ -4,8 +4,8 @@ services:
   web:
     tty: true
     build: .
-    command: bash -c "yarn && yarn start"
+    command: 'yarn start'
     ports:
-      - '8080:8080'
+      - "8080:8080"
```

Comment:
I believe they both work. I came across [this]http://stackoverflow.com/questions/19109912/do-i-need-quotes-for-strings-in-yaml) SO question/answer when browsing between the difference. However, I ultimately decided on double quotes because in Docker's own documentation they use double. See [here](https://docs.docker.com/compose/compose-file/#expose)